src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -129,7 +147,7 @@ export const useTradeRoutes = (
 
   useEffect(() => {
     setDefaultAssets()
-  }, [connectedChainId, setDefaultAssets])
+  }, [connectedEvmChainId, setDefaultAssets])
```

Comment:
What about non-EVM ChainIds?  Should we abstract the `supportsXYZ` and add them as dependencies?

Reply:
We need to tread with caution here. We have an ugly set of cascading re-renders across the swapper, and adding more dependencies here, even though it may well be "correct" to do so, amplifies the issue.

I spent a couple of hours wrangling it today, and am going to park it for now.